aikau/src/main/resources/alfresco/layout/FixedHeaderFooter.js

```diff
@@ -277,8 +270,8 @@ define(["alfresco/core/ProcessWidgets",
        *
        * @instance
        */
-      _resize: function alfresco_layout_FixedHeaderFooter___resize() {
-         if (this.recalculateAutoHeightOnResize === true)
+      onResize: function alfresco_layout_FixedHeaderFooter__onResize() {
```

Comment:
But that's exactly what is happening... this is called when something has resized, e.g. the window has resized, or an enclosing widget has resized - this in turn triggers the FixedHeaderFooter to resize.